parquet-hadoop/src/main/java/parquet/hadoop/MemoryManager.java

```diff
@@ -39,16 +40,19 @@
 public class MemoryManager {
   private static final Log LOG = Log.getLog(MemoryManager.class);
   static final float DEFAULT_MEMORY_POOL_RATIO = 0.95f;
+  static final long DEFAULT_MIN_MEMORY_ALLOCATION = 0;
```

Comment:
We could have a better minimum than 0 which is in any case a bad value.
Possibly something related to the pageSize or the initial blockSize?